adivinar.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>BIENVENDO A MI NUEVA PAGINA</title>
+</head>
+
+
+<body>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta nombre="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="adivinar.css"/>
+<h1>ADIVINADOR</h1>
+<br>
+<button type="submit" id="generar-numero">Generar numero aleatorio</button>
+<p>Numero de intentos restantes:<strong id="intentos"></strong></p>
+<label>Introduce un numero:</label>
```

Comment:
Los label deben llevar un atributo for="" con el id del input al que hacen referencia. En este caso sería 
<label for="numero">Introduce un numero:</label>

Reply:
gracias! modificado :)

Reply:
Fijate tambien que podes intentar generar un unico div para desplegar el resultado, y con un condicional elegir cual va a ser la propiedad "innerText" de tal div. Si fue correcta la adivinanza, será un mensaje A (Programado dentro de javascript), e lo contrario, sera un mensaje B. Así ahorras líneas y elementos de HTML.